java/messages/src/main/java/com/nedap/retail/messages/Client.java

```diff
@@ -83,7 +83,7 @@ public void destroy() {
      * System API: list
      */
     public List<SystemListPayload> getSystemList() {
-        final WebResource resource = resource("/system/1.0/list");
+        final WebResource resource = resource("/system/v1/list");
```

Comment:
I saw that we changed endpoint `1.0 -> v1` so this is probably good change but will we have backward compatibility issues, is this method used?

Maybe someone more familiar with this endpoint @jasperkremer @bmjansen ?